src/components/WordGroupSelection/WordGroupSelection.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import {collectedWordGroupsService} from "util/GWUtil/resource";
+import "./WordGroupSelection.css";
+import WordGroupIcon from "./WordGroupIcon";
+import WordSelection from "./WordSelection";
+
+function WordGroupSelection(props) {
```

Comment:
corresponding w angela's component names could you rename this + the other variables to `selector` or `picker`?